assets/js/adminCtrl.js

```diff
@@ -83,8 +83,54 @@ define([
             });
         };
 
+        $scope.getFlairs = function () {
```

Comment:
We do, but it's in userCtrl.js. I wasn't sure if there was an easy way to share the functions between the clientside controllers.